cmd/activator/request_log.go

```diff
@@ -52,6 +52,8 @@ func requestLogTemplateInputGetter(revisionLister servinglisters.RevisionLister)
 	return func(req *http.Request, resp *pkghttp.RequestLogResponse) *pkghttp.RequestLogTemplateInput {
 		namespace := req.Header.Get(activator.RevisionHeaderNamespace)
 		name := req.Header.Get(activator.RevisionHeaderName)
+		name, namespace = activator.GetNameAndNamespace(name, namespace, req.Host)
```

Comment:
another option, since we have the `req` here, is to grab the revision from the context (since the other place this code is used is to put the revision _in_ the context in context_handler.go :)). That way we could delete the duplicated revisionLister.Get below too. What do you think about that approach?